test/e2e/changedFilesUtil.js

```diff
@@ -30,15 +30,16 @@ async function readChangedFiles() {
  * @returns {Promise<string[]>} An array of filtered E2E test file paths.
  */
 async function filterE2eChangedFiles() {
-  const changedFiles = await readChangedFiles();
-  const e2eChangedFiles = changedFiles
-    .filter(
-      (file) =>
-        file.startsWith('test/e2e/') &&
-        (file.endsWith('.spec.js') || file.endsWith('.spec.ts')),
-    )
-    .map((file) => `${BASE_PATH}/${file}`);
-  return e2eChangedFiles;
+  return []; // temp for experimenting
```

Comment:
I don't intend to check this in.  But the feature to run changed tests extra times is exceeding the 20 minute timeout.  I suggested increasing it here: https://github.com/MetaMask/metamask-extension/pull/27146